graphql/src/store/query.rs

```diff
@@ -297,14 +337,23 @@ fn build_order_by(
     }
 }
 
+/// Kamil: I don't like it... What if we have `_` in the field name? For now I will use double underscore
+fn parse_field_as_order(field_name: &String) -> (&str, Option<&str>) {
+    let mut field_name_parts = field_name.split("__");
```

Comment:
Used double underscore for now

Reply:
For now, double underscore is fine, but another option would be to use an input object: `orderBy: { child_: field }` but I think that runs into the same problem we had elsewhere that input types can't be unions (since we still need `orderBy: parentField`) A third option would be to have a separate `orderByChild: field`, i.e. for a `books` subfield you'd write `orderByBooks: title`